app/scripts/modules/core/src/diffs/viewChangesLink.component.ts

```diff
@@ -7,7 +8,20 @@ import {COMMIT_HISTORY_COMPONENT} from './commitHistory.component';
 import {EXECUTION_SERVICE, ExecutionService} from 'core/delivery/service/execution.service';
 import {JAR_DIFF_COMPONENT, IJarDiff} from './jarDiff.component';
 
+export interface IJenkinsInfo {
+  host: string;
+  name: string;
+  number: string;
+}
+
+export interface IBuildInfo {
+  ancestor: string,
+  jenkins?: IJenkinsInfo;
+  target: string,
+}
```

Comment:
That's a good point.  This actually belongs on `IStage` since `ancestor` and `target` are the new fields I added to the new `buildInfo` property on the stage context in orca.  I'll move to there.